src/test/java/uk/gov/hmcts/reform/pip/data/management/service/PublicationServiceTest.java

```diff
@@ -1180,4 +1192,26 @@ void testMaskEmailEmptyString() {
                      "Email was not masked correctly");
     }
 
+    @Test
+    void testMiService() {
+        when(artefactRepository.getMiData()).thenReturn(EXAMPLE_CSV);
+        String testString = publicationService.getMiData();
+        String[] splitLineString = testString.split("\r\n|\r|\n");
+        long countLine1 = splitLineString[0].chars().filter(character -> character == ',').count();
+        assertThat(testString)
+            .as("Header row missing")
+            .contains("source_artefact_id");
+        assertThat(splitLineString)
+            .as("Only one line exists - data must be missing, as only headers are printing")
+            .hasSizeGreaterThanOrEqualTo(2);
+        assertThat(splitLineString)
+            .as("Wrong comma count compared to header row!")
+            .allSatisfy(
+                e -> assertThat(e.chars().filter(character -> character == ',').count()).isEqualTo(countLine1));
+        String testString2 = publicationService.getMiData();
```

Comment:
This testString2 is not the same as testString? Can we not simply use testString in the assertion below?

Reply:
Yep - sorry, this is an artifact from when I had two separate tests before merging. Have fixed now.